radix-engine-common/src/constants/transaction_construction.rs

```diff
@@ -0,0 +1,2 @@
+/// The default tip percentage to use when building a transaction.
```

Comment:
I feel like in tests we may wish to default to this? But the actual transaction builder should default to 0?

Reply:
Yes, let me change it to zero.